packages/cloudvision-grpc-web/rollup.config.js

```diff
@@ -14,6 +14,12 @@ const projectRootDir = path.resolve(__dirname);
 const config = {
   input: 'src/index.ts',
   onwarn: (warning) => {
+    if (warning.code === 'CIRCULAR_DEPENDENCY') {
```

Comment:
Why are you ignoring these?

Reply:
protobufjs has a circular dependency in the generated code. It's pretty harmless but blocks rollup from compiling